### DISCUSSION TITLE
Unable to retrieve just added module and its data.

### DISCUSSION THREAD

**Discussion by UmerTariqKhan** [mixcore/mix.core#400] [Q&A] [answered]
Hi, I am unable to fetch module data. following are steps what I am doing.. Any help will be appreiciated.
1- Add a new Theme and Activate it.
2- Set Master -> layout to its js/CSS files.
3- Add Module template then add module
4- Trying to fetch module data to show, but always get null.

Thanks

![image](https://user-images.githubusercontent.com/36357077/122135815-ad038680-ce5a-11eb-8e89-324b7884fb3c.png)
![image](https://user-images.githubusercontent.com/36357077/122135435-f43d4780-ce59-11eb-8dde-e036ce76750e.png)

**Comment by Smilefounder** [comment, marked as answer]
Hi @UmerTariqKhan I think we missed this one. Did you resolve the issue yet?
